examples/cassandra/README.md

```diff
@@ -300,47 +226,106 @@ spec:
+image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
+This is a standard Cassandra installation on top of Debian.  However, it also
+adds a custom
+[`SeedProvider`](https://svn.apache.org/repos/asf/cassandra/trunk/src/java/org/apache/cassandra/locator/SeedProvider.java) to Cassandra.  In
+Cassandra, a ```SeedProvider``` bootstraps the gossip protocol that Cassandra
+uses to find other nodes.
+The [`KubernetesSeedProvider`](java/src/io/k8s/cassandra/KubernetesSeedProvider.java)
+discovers the Kubernetes API Server using the built in Kubernetes
+discovery service, and then uses the Kubernetes API to find new nodes.
+See the [image](image/) directory of this example for specifics on
+how the container image was built and what it contains.
 
-Create this controller:
+You may also note that we are setting some Cassandra parameters (`MAX_HEAP_SIZE`
+and `HEAP_NEWSIZE`), and adding information about the
+[namespace](../../docs/user-guide/namespaces.md).
+We also tell Kubernetes that the container exposes
+both the `CQL` and `Thrift` API ports.  Finally, we tell the cluster
+manager that we need 0.1 cpu (0.1 core).
+
+Create the Replication Controller:
 
 ```console
 $ kubectl create -f examples/cassandra/cassandra-controller.yaml
 ```
 
-Now this is actually not that interesting, since we haven't actually done
-anything new.  Now it will get interesting.
-
-Let's scale our cluster to 2:
+You can list the new controller:
 
 ```console
-$ kubectl scale rc cassandra --replicas=2
+$ kubectl get rc -o wide
+NAME        DESIRED   CURRENT   AGE       CONTAINER(S)   IMAGE(S)                             SELECTOR
+cassandra   2         2         11s       cassandra      gcr.io/google-samples/cassandra:v8   app=cassandra
 ```
 
-Now if you list the pods in your cluster, and filter to the label `app=cassandra`, you should see two cassandra pods:
+Now if you list the pods in your cluster, and filter to the label
+`app=cassandra`, you should see two Cassandra pods. (The `wide` argument lets
+you see which Kubernetes nodes the pods were scheduled onto.)
 
 ```console
-$ kubectl get pods -l="app=cassandra"
-NAME              READY     STATUS    RESTARTS   AGE
-cassandra         1/1       Running   0          3m
-cassandra-af6h5   1/1       Running   0          28s
+$ kubectl get pods -l="app=cassandra" -o wide
+NAME              READY     STATUS    RESTARTS   AGE       NODE
+cassandra-21qyy   1/1       Running   0          1m        kubernetes-minion-b286
+cassandra-q6sz7   1/1       Running   0          1m        kubernetes-minion-9ye5
 ```
 
-Notice that one of the pods has the human-readable name `cassandra` that you
-specified in your config before, and one has a random string, since it was named
-by the replication controller.
+Because these pods have the label `app=cassandra`, they map to the service we
+defined in Step 1.
+
+You can also query the service endpoints to check if the pod has been correctly
```

Comment:
nit: Will be easier to parse this if we start with the goal and follow up with the how.

e.g.
Check that the Pod is visible to the Service using the following service endpoints query (substituting ....)